components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add-finish-ajaxprocessor.jsp

```diff
@@ -21,6 +21,7 @@
 <%@ page import="org.owasp.encoder.Encode" %>
 <%@ page import="org.wso2.carbon.CarbonConstants" %>
 <%@ page import="org.wso2.carbon.identity.core.util.IdentityUtil" %>
+<%@ page import="org.wso2.carbon.identity.oauth2.internal.OAuth2ServiceComponentHolder" %>
```

Comment:
check whether this can be set serverwide and read everywhere or whether we need to go with the componenet holder.